### ISSUE TITLE
[Enhancement] Accessibility improvements for contrast ratio

### ISSUE THREAD

**Issue by OOlashyn** [PowerPlatformToolBox/pptb-web#61]
Across multiple pages, we have an insufficient contrast ratio (validated via Chrome Lighthouse). To address that, we need to update some colors in our color scheme and when we are using them.

Example from Versions page

<img width="752" height="724" alt="Image" src="https://github.com/user-attachments/assets/3e5d934f-74a3-400c-a2fd-153f2735bfe7" />
